src/uct/ib/base/ib_md.h

```diff
@@ -157,6 +157,12 @@ typedef struct uct_ib_md {
         uint32_t             base;
         uint32_t             size;
     } mkey_by_name_reserve;
+    struct {
+        uint8_t              rc_rdma;
+        uint8_t              dc_rdma;
+        uint8_t              xrc_rdma;
```

Comment:
why do we need config for xrc?

Reply:
thanks, removed, I was assuming xrc == rc_x somehow.